Documentation/Books/Manual/Deployment/StandaloneAgency/README.md

```diff
@@ -22,12 +22,12 @@ Clearly all cases, which would form disjunct subsets of agents would break or in
 
 Typically, one achieves fairly high fault-tolerance with low, odd number of agents while keeping the necessary network traffic at a minimum. It seems that the typical agency size will be in range of 3 to 7 agents.
```

Comment:
Maybe easier to list all three options, given that it should be an odd number. Is an even number even possible? The text doesn't clearly state that it *must* be an odd number...

```suggestion
Typically, one achieves fairly high fault-tolerance with low, odd number of agents while keeping the necessary network traffic at a minimum. It seems that the typical agency size will be 3, 5 or 7 agents.
```
Also, "it seems" sounds rather vague. How about saying "It should typically be ..."?